tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/HttpMessageHandlerTests.cs

```diff
@@ -77,6 +77,11 @@ public HttpMessageHandlerTests(ITestOutputHelper output)
             string metadataSchemaVersion,
             bool traceId128Enabled)
         {
+            if (EnvironmentHelper.IsCoreClr() && EnvironmentHelper.GetTargetFramework() == "netcoreapp2.1")
+            {
+                throw new SkipException("This test is flaky on netcoreapp 2.1");
+            }
```

Comment:
Instead of skipping it entirely, we could consider catching the error and throwing a `SkipException`. That way we know it works _sometimes_ on .NET Core 2.1 at least (with the suggested approach we would have no idea if we completely broke it)

Reply:
Yes I thought of it and chose the lazy way. I can come back to it later.

Reply:
Fixed in 76ae783c3748cc5d0d184483a64626576ea6e4c2